src/state/actions/query.js

```diff
@@ -157,3 +160,45 @@ export const executeQuery = (updateUrlParams = true) => {
     }
   };
 };
+
+export const updateSuggestions = suggestionString => {
+  return async dispatch => {
+    dispatch({
+      type: SET_SUGGESTION_STRING,
```

Comment:
I have the feeling that you don't need to store in the `query` state the suggestion string. We use the state to persist all user selection, in the sense that if you for example modify the sort by or pagination, we want to keep the same query and sort order previously selected by the user.
I think here you perform always a query when asking for suggestion, you don't need any other previous or other value, no?

But again I might be wrong! :)

Reply:
As discussed IRL it is usefull to save to redux e.g. in case of filtering by aggregations.